client/components/single-product-plan/README.md

```diff
@@ -0,0 +1,39 @@
+Single Product Plan
+=======
+
+Single Product Plan is a React component for rendering a box with single product short description and purchase options.
+
+It's used e.g. on `my-plans` page near the bundle plans grid. 
+
```

Comment:
Also, let's document any additional behavior that is not obvious, like:

* When a product is purchased, it will display the subtitle. When its not purchased, it will display the price/price range, etc.

Reply:
Sounds good, I'll add some more details to the readme file.